common/persistence/sql/execution.go

```diff
@@ -145,13 +146,21 @@ func (m *sqlExecutionStore) createWorkflowExecutionTx(
 					Msg: fmt.Sprintf("Workflow execution creation condition failed. WorkflowId: %v, "+
 						"LastWriteVersion: %v, PreviousLastWriteVersion: %v",
 						workflowID, row.LastWriteVersion, request.PreviousLastWriteVersion),
+					RequestID:        row.CreateRequestID,
+					RunID:            row.RunID.String(),
+					State:            row.State,
```

Comment:
oh, did we store decoded version of state for SQL?

Reply:
yep